src/app/static/src/tests/testHelpers.ts

```diff
@@ -67,33 +70,36 @@ export function expectAllMutationsDefined(mutationDefinitions: any, mutationTree
     }
 }
 
-export function expectTranslatedWithStoreType<T extends TranslatableState>(element: DOMWrapper<any>,
+export async function expectTranslatedWithStoreType<T extends TranslatableState>(element: DOMWrapper<any>,
                                                                            englishText: string,
                                                                            frenchText: string,
                                                                            portugueseText: string,
                                                                            store: Store<T>,
                                                                            attribute?: string) {
+    const value = () => attribute ? element.attributes(attribute) : element.text();
```

Comment:
Why is this a function ?

Reply:
not sure, I have not actually changed this at all from when this was originally implemented, can do an opportunistic commit to clean that up!

Reply:
ahh just tried it and realised why! we need this to be a function because we only want to get the value once the state has updated, if this wasn't a function then it would store the english string in value and when we change the language to french, value will not update!